.github/workflows/generate-cred.yml

```diff
@@ -0,0 +1,24 @@
+name: generate-cred
+
+on:
+  pull_request: []
+  schedule:
+    # Nightly at 2:30am 
+    - cron: 30 2 * * *
+
+jobs:
+  GenerateCred:
+    name: Generate Cred
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v1
+      - name: Run SourceCred
+        uses: vsoch/cred-action@master
```

Comment:
Let’s not pin this to `master`, which introduces nondeterminism in
upstream projects and also makes security auditing impossible. Pinning
to an explicit commit hash is best; a release tag is also okay.

Reply:
This (along with having pull_requests: []) to trigger was primarily for testing so that I could push to master and then test via the pull request. For some final recipe, this would be removed, and a commit / verison tag used.

Reply:
okay, just updated to be for a commit (the one just merged to remove jq)